### PR TITLE
adding page to guide for semantic html

### PR DESCRIPTION
**Pull request by jaube05**

### CODE REVIEW

File: _data/guide.yml [hunk 1]
```diff
@@ -61,6 +61,11 @@ docs:
   description: 
   icon: <i class="far fa-file-alt fa-5x" aria-hidden="true"></i>
 
+- title: Semantic HTML
+  url: /semantic-html
+  description:
+  icon: <i class="far fa-code fa-5x" aria-hidden="true"></i>
```
Comment (dmundra) [inline _data/guide.yml:67]: @jaube05 fa-code for some reason didn't generate an icon https://accessibility-qa.civicactions.com/guide/, maybe choose a different one?

**Review by dmundra — DISMISSED** [review]
@jaube05 other than the icon the page looks good.

File: _data/guide.yml (hunk 1 above)
Comment (jaube05) [reply _data/guide.yml]: Done

**Review by dmundra — APPROVED** [review]
@jaube05 looking great. Thank you!

**Pull request merged**